virtual-desktop/src/app/window-manager/mvd-window-manager/launchbar/launchbar-widget/launchbar-widget.component.ts

```diff
@@ -22,20 +22,27 @@ import {
   } from '@angular/core';
 import { Observable } from 'rxjs/Rx';
 
+import { LanguageLocaleService } from '../../../../shared/language-locale.service';
+
 @Component({
   selector: 'rs-com-launchbar-widget',
   templateUrl: 'launchbar-widget.component.html',
-  styleUrls: [ 'launchbar-widget.component.css' ]
+  styleUrls: [ 'launchbar-widget.component.css' ],
+  providers: [LanguageLocaleService]
 })
 export class LaunchbarWidgetComponent implements OnInit {
   date: Date;
   popupVisible: boolean;
   @Output() popupStateChanged = new EventEmitter<boolean>();
   @ViewChild('usericon') userIcon: ElementRef;
   @ViewChild('logoutbutton') logoutButton: ElementRef;
+  @ViewChild('languagebutton') languageButton: ElementRef;
```

Comment:
Let's comment these out with a note that we do need to add these once we get the right UI design.